spec/datasets/djornl/edge_type.yaml

```diff
@@ -52,3 +52,20 @@ oneOf:
     title: ATRM TF to Target LitCurated 01082020 TranscriptionFactorToGene
     description: Contains literature mined and manually curated TF regulatory interactions for A.thaliana from 1701 TFFs from PlantTFDB 2.0 and 4663 TF-associated interactions. These were manually filtered (e.g. FPs, PPI interactions removed). They then added some from other sources. Downloaded from http://atrm.cbi.pku.edu.cn/download.php
 
+  - const       : 'Metabolic_AraCyc_Gene_to_Gene'
+    title       : 'AraCyc_Gene_to_Gene'
+    description : "AraCyc.  GeneA connects to GeneB if they are both enzymes and are linked by a common substrate or product. E.g. RXNA (GeneA) → Compound1 → RXNB (GeneB). Here GeneA connects to GeneB due to Compound1.  Date accessed 03/18/2021.  All enzymatic reactions and associated genes (enzymes) and compounds were downloaded from AraCyc as smarttables, and transformed by custom R script to generate the appropriate network format linking gene to gene based on intermediary compounds.  Bringing to-spec, 322 non-standard edges were removed and 3,243 duplicates were removed."
+  - const       : 'GO'
+    title       : 'Athal_GOSemSim_BP_WangResCombined.score0.3'
+    description : "GO networks are Gene-Gene networks where an edge indicates that two genes are similar to each other based on shared GO terms.  D.Kainer used the GoSemSim R package to get pairwise similarity between all GO annotated genes.  Only GO annotations with solid evidence codes were used. The following GO codes were dropped: badcodes <- c(IEA,ISS,NAS,ND,RCA) Two Semantic Similarity measures were used: 1. Wang: a graph-based measure 2. Resnik: a simple Information Content based measure.  The average was used as the final score."
```

Comment:
> Gene-Gene
no need to capitalise "Gene"

> based on shared GO terms
strictly speaking, should be "based on shared GO annotations"

> The following GO codes were dropped: badcodes <- c(IEA,ISS,NAS,ND,RCA)
this looks like it was lifted from a script -- can you turn it into English?

> Two Semantic Similarity measures were used: 1. Wang: a graph-based measure 2. Resnik: a simple Information Content based measure.
no need to capitalise 'Semantic Similarity' or 'Information Content'